spec/std/ini_spec.cr

```diff
@@ -34,4 +34,10 @@ describe "INI" do
       })
     end
   end
+
+  describe "build from Hash(String, Hash(String, String)" do
+    it "build file" do
+      INI.build(INI.parse(File.read "#{__DIR__}/data/test_file.ini"), " ").should eq(File.read "#{__DIR__}/data/test_file.ini")
```

Comment:
You should add an actual example in code which does not rely on the `INI.parse` and the contents of an additional file. The example in the docstring would do.